server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node dist/index.js",
+    "start": "concurrently \"react-scripts start\" \"cd server && nodemon server\"",
```

Comment:
@miabu-pashh What command did you use to start up the server? This change is not related to this issue. If you want to update any configs or startup scripts, create an another PR for that. DO NOT include all of them in a single PR. Please revert this file. Please only commit the relevant files to your branch.